src/ui/popup/style.less

```diff
@@ -26,10 +26,6 @@ body {
     padding: @popup-content-padding;
     position: relative;
     width: @popup-content-width;
-    // Prevent popup from entering a feedback loop when it displays scrollbars,
-    // adjusts popup size to be larger, then hides scrollbars, adjusts popup size to be smaller,
-    // then displays scrollbars, etc.
-    overflow: hidden;
```

Comment:
This line was causing #10178, but I decided to revert all CSS changes which could in theory cause other regressions too.